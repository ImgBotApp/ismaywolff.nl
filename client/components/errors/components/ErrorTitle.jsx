import styled from 'styled-components'

const ErrorContainer = styled.h2`
  margin: 0;
  text-align: center;
`

export default ErrorContainer
