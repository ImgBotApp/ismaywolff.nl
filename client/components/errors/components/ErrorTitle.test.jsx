import React from 'react'
import { shallow } from 'enzyme'
import { matcher, serializer } from 'jest-glamor-react'
import ErrorTitle from './ErrorTitle'

expect.addSnapshotSerializer(serializer)
expect.extend(matcher)

describe('<ErrorTitle />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ErrorTitle />)
    expect(wrapper).toMatchSnapshotWithGlamor()
  })
})
