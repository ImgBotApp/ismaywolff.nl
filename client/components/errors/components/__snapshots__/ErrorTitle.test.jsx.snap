// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorTitle /> renders correctly 1`] = `
.css-1iby2t6,
[data-css-1iby2t6] {
  margin: 0;
  text-align: center;
}

<h2
  className="css-1iby2t6"
/>
`;
